local-cluster/bench/Main.hs

```diff
@@ -33,8 +35,11 @@ data Options = Options
   , concurrency :: Int
   , timeoutSeconds :: DiffTime
   , clusterSize :: Word64
+  , constantUtxo :: GeneratorType
   }
 
+data GeneratorType = LargeTxs | ConstantUtxo
```

Comment:
`LargeTxs` -> "standard generator" provided by ledger-specs, leading to an ever growing UTXO set because -> `GrowingUtxo`